index.js

```diff
@@ -0,0 +1,532 @@
+'use strice'
```

Comment:
one of the best idea is to split files.
one is used for view part of application another part for logic part